functions.php

```diff
@@ -220,7 +220,7 @@ function ( $hidden ) {
 add_filter(
 	'block_editor_settings_all',
 	function ( array $editor_settings, WP_Block_Editor_Context $block_editor_context ) {
-		if ( 'post' !== $block_editor_context->post->post_type ) {
+		if ( isset( $block_editor_context->post ) && 'post' !== $block_editor_context->post->post_type ) {
```

Comment:
This is not really related to the ticket, but it's such a small change that I didn't think it required a separate PR